src/topology/local_homeomorph.lean

```diff
@@ -768,6 +768,23 @@ h.to_local_equiv_target
 
 end open_embedding
 
+-- We close and reopen the namespace to avoid
+-- picking up the unnecessary `[nonempty α]` typeclass argument
+namespace open_embedding
+
+lemma continuous_at_iff
```

Comment:
Should this be `@[simp, mfld_simps]`?  Not sure, just wondering.

Reply:
I didn't put `simp` on it because it seemed unlikely that `hf : open_embedding` would be solved by `simp` unless it was provided manually to `simp`.

On the other hand, I like `simp` lemmas, so I'm happy to do this.

Reply:
I am really not sure, so please make your own call :)  Or ping sgo****l (censored to prevent auto-ping) for a third opinion.